src/databases/Rect/avtRectFileFormat.C

```diff
@@ -29,27 +29,16 @@
 
 #include <BadIndexException.h>
 #include <DebugStream.h>
+#include <FileFunctions.h>
 #include <InvalidVariableException.h>
 #include <InvalidFilesException.h>
 
-#include "visit_gzstream.h"
+#include <visit_gzstream.h>
 
 
 using std::vector;
 using std::string;
 
-static string GetDirName(const char *path);
-
-#if defined(_WIN32)
-// Define >> for strings since Windows does not seem to have it.
-ifstream &operator >> (ifstream &s, string &str)
```

Comment:
This isn't an issue for Windows anymore?

Reply:
No. That's why it was removed.